templates/up.sh.j2

```diff
@@ -5,3 +5,4 @@ brctl addif {{ openvpn_bridge_dev }} {{ openvpn_dev }}; true
 brctl addif {{ openvpn_bridge_dev }} {{ openvpn_interface }}; true
 ip l set {{ openvpn_bridge_dev }} up;
 ip l set {{ openvpn_dev }} up;
+ip a r {{ openvpn_bridge.address }} dev {{ openvpn_bridge_dev }}; true
```

Comment:
You could write these commands in full if you touch it next time. In a script it doesn't matter if you write `l` or `link` but the reader may not know all the abbreviations.